internal/cli/atlas/auditing/auditing.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auditing
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "auditing",
+		Short: "Returns and edits database auditing settings for MongoDB Cloud projects.",
```

Comment:
```suggestion
		Short: "Returns database auditing settings for MongoDB Cloud projects.",
```
maybe add back "and edits" when those edit commands are added?